coralproject/talk
# [8.6.5] Release `develop` into `main`

## What does this PR do?

- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468
- https://github.com/coralproject/talk/pull/4428

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

See:
- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468

## Does this PR introduce any new environment variables or feature flags?

See:
- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468

## If any indexes were added, were they added to `INDEXES.md`?

See:
- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468

## How do I test this PR?

See:
- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468

## Were any tests migrated to React Testing Library?

See:
- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468

## How do we deploy this PR?

See:
- https://github.com/coralproject/talk/pull/4313
- https://github.com/coralproject/talk/pull/4468
